media-sound/fluid-soundfont/fluid-soundfont-3.1.ebuild

```diff
@@ -1,29 +1,62 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-DESCRIPTION="The Fluid R3 soundfont"
-HOMEPAGE="http://musescore.org/en/handbook/soundfont"
+# There are no new releases for old data files but the archival repo can update
+# the accompanying documentation [but this ebuild only cares for the data ;) ].
+COMMIT=700c7c1775da12f630750cc34149763ee52aa6c6
+
+DESCRIPTION="The Fluid R3 version 1 soundfont"
+HOMEPAGE="https://github.com/pinkflames/fluid-soundfont"
 SRC_URI="
-	http://prereleases.musescore.org/soundfont/${PN}_${PV}.tar.gz
-	timidity? ( https://dev.gentoo.org/~hwoarang/distfiles/timidity.cfg.bz2 )"
+	https://github.com/pinkflames/${PN}/archive/${COMMIT}.tar.gz -> ${PN}-${COMMIT}.tar.gz
+"
 
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="amd64 ~ppc64 x86"
```

Comment:
FYI, not really comfortable pushing an update like this to a stable ebuild. Please reset the KEYWORDS to ~ and revbump your work.

Reply:
I understand. It's mostly a remnant from when I was trying to use GitHub's Large File Storage but had to go with the FLACy hack, because it would never have worked with the Gentoo mirror traffic alone, let alone any users accessing the files directly.